aws/resource_aws_db_instance.go

```diff
@@ -404,6 +404,25 @@ func resourceAwsDbInstance() *schema.Resource {
 				},
 			},
 
+			"performance_insights_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+
+			"performance_insights_kms_key_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validateArn,
+			},
+
+			"performance_insights_retention_period": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
```

Comment:
What is the default retention period if PI is enabled? Instead of using `Computed: true` here we likely need to configure Terraform to ignore the difference of the default value when its enabled.

```go
// Example DiffSuppressFunc if PI defaults to 7 when enabled, otherwise we expect 0 (undefined from API)
DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
	if !d.Get("performance_insights_enabled").(bool) {
		return false
	}
	if old == "7" && new == "0" {
		return true
	}
	return false
},
```